home.html

```diff
@@ -0,0 +1,128 @@
+                    <figure class="cancel" alt="">
+                        <i class="fa fa-times"></i>
+                    </figure>
+                    <ul class="menu-list">
+                        <li class="menu-item"><a href="#front"><h4>Home</h4></a></li>
+                        <li class="menu-item"><a href="./about.html"><h4>About</h4></a></li>
+                        <li class="menu-item"><a href="#program"><h4>Program</h4></a></li>
+                        <li class="menu-item"><a href="#partner"><h4>Partners</h4></a></li>
+                    </ul>
+                </div>
+            </div>
+        </header>
+        <section id="front" class="container">
+            <div class="container" id="hello">
+                <h5>#Sadico</h5>
+                <h1>WELCOME TO
+                    <br>
+                    RAWAYANA'S LATIN  
+                    <br>
+                    AMERICA TOUR 2023
+                </h1>
+            </div>
+
+            <div class="container" id="intro">
+                <p> 
+                    In 2022, Rawayana continued to tour internationally in support of their latest critically acclaimed album, 
+                    "Cuando Los Acéfalos Predominan", performing at major festivals.
+                </p>
+            </div>
+
+            <div class="container" id="schedule">
+                <h2>2023.08.13(THU) ~ 14(FRI)</h2>
+                <h3>@ La Concha Acustica de Andres Bello</h2>
+            </div>
+        </section>
+
+        <section class="container" id="program">
+            <h2> Concert Schedule </h2>
+            <hr>
+
+            <div id="cards">
+
+            </div>
+
+            <button class="container" id="join">
+                Get your tickets today!
+            </button>
+
+            <h3>SEE THE WHOLE SCHEDULE</h3>
+        </section>
+
+        <section class="container" id="featured">
+            <h2> Featured Supporters </h2>
+            <hr>
+
+            <div id="speakers">
+                
+            </div>
+
+            <button class="container" id="more">
+                MORE
+            </button>
+
+        </section>
+        
+        <section class="container" id="partner">
+            <h3>Partner</h3>
+            <hr>
+            <div class="container">
+                <img src="./Assets/partners/mozilla.png"/>
+                <img src="./Assets/partners/google.png"/>
+                <img src="./Assets/partners/naver.png"/>
+                <img src="./Assets/partners/daumkakao.png"/>
+                <img src="./Assets/partners/airbnb.png"/>
+            </div>
+        </section>
+
+        <footer class="contianer" id="cc">
+            <div id="logo">
+                <img src="/Assets/logo.png">
+            </div>
+            <div id="rights">
+                <p>
+                    2023 Creative Commons. <br>
+                    Some rights reserved.
+                </p>
+            </div>
+        </footer>
```

Comment:
- [ ] kindly consider increasing the size of this footer from top and bottom as you footer size is smaller than the footer on behance design.